vispy/visuals/tests/test_axis.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you copy the header from the other modules and add it here?

Reply:
Done